transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/HyperwalletTransferMethodUi.java

```diff
@@ -131,6 +133,26 @@ public Intent getIntentAddTransferMethodActivity(@NonNull final Context context,
         return intent;
     }
 
+    /**
+     * @param context              A Context of the application consuming this Intent.
+     * @param transferMethodType   The type of transfer method. For a complete list of transfer methods, see {@link
+     *                             com.hyperwallet.android.model.transfermethod.TransferMethod.TransferMethodTypes}
+     * @param transferMethodToken  The Transfer method token.
+     * @param lockScreenToPortrait if set <code>true</code> screen will be locked to Portrait mode;
+     *                             otherwise <code>false</code> screen will follow whatever the
+     *                             device orientation is directed.
+     * @return an Intent with the data necessary to launch the {@link UpdateTransferMethodActivity}
+     */
+    public Intent getIntentUpdateTransferMethodActivity(@NonNull final Context context,
+            @NonNull final String transferMethodType, @NonNull final String transferMethodToken,
+            final boolean lockScreenToPortrait) {
+        Intent intent = new Intent(context, UpdateTransferMethodActivity.class);
+        intent.putExtra(EXTRA_TRANSFER_METHOD_TYPE, transferMethodType);
```

Comment:
This is wrong, integrator will only pass transferMethodToken, as I explained in planning transferMethodType you will need to get from graphql response